app/views/play/CampaignView.coco.coffee

```diff
@@ -1338,6 +1339,19 @@ module.exports = class CampaignView extends RootView
     @openModalView new SubscribeModal()
     window.tracker?.trackEvent 'Show subscription modal', category: 'Subscription', label: 'campaignview premium button'
 
+  onClickM7OffButton: (e) ->
+    noty({ text: $.i18n.t('play.confirm_m7_off'), layout: 'center', type: 'warning', buttons: [
+      { text: 'Yes', onClick: ($noty) =>
+        if me.getM7ExperimentValue() == 'beta'
+          me.updateExperimentValue('m7', 'control')
+          $noty.close()
+          @render()
+      }, { text: 'No', onClick: ($noty) -> $noty.close() }]
+    })
+
+
+
```

Comment:
remove extra lines

Reply:
yes, it should be ["Something really unobtrusive that most players won't notice unless directed to."](https://app.asana.com/0/0/1202317318282996/1202823842099215/f)
extra lines removed